_posts/2020-12-17-security-hub.md

```diff
@@ -0,0 +1,319 @@
+
+Now, the first step to using Security Hub is to elect the admin account and then enable all other accounts as members. It consolidates findings into the admin account for each region to give a centralised view of your security state.
+
+The method we used to rollout Security Hub at the time was the [Security Hub Enabler](https://github.com/aws-samples/aws-control-tower-securityhub-enabler) tool provided by AWS. As of November 2020, Security Hub now supports [AWS Organizations](https://aws.amazon.com/organizations/), which is a much simpler method for enabling Security Hub.
+
+To [elect the Security Hub admin account](https://docs.aws.amazon.com/cli/latest/reference/securityhub/enable-organization-admin-account.html), run this via the AWS CLI for your Organization root account (where `123456789012` is the account ID you want to elect) in each of the regions you want to use Security Hub:
+
+```sh
+aws securityhub enable-organization-admin-account --admin-account-id 123456789012
+```
+
+According to the [documentation](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-accounts-orgs.html), this will also add all AWS Organizations accounts as Security Hub members.
+
+Then, from your Security Hub main account, run the following to [automatically add new accounts](https://docs.aws.amazon.com/cli/latest/reference/securityhub/update-organization-configuration.html) as Security Hub members in the future:
+
+```sh
+aws securityhub update-organization-configuration --auto-enable
+```
+
+That's it! You should now have the [CIS AWS Foundations benchmark](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards-cis.html) and [AWS Foundational Security Best Practices standard](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards-fsbp.html) packs enabled in all accounts by default.
+
+## Compliance pack review
+
+Once you have enabled Security Hub, we recommend reviewing the individual security controls for each of the compliance packs:
+
+- [CIS AWS Foundations Benchmark controls](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cis-controls.html)
+- [AWS Foundational Security Best Practices controls](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards-fsbp-controls.html)
+
+Not all controls will be relevant to your organisation, also there is some overlap between the two default packs. Also, we should enable rules that apply to global services in one region (e.g. IAM rules).
+
+An example of a rule that we disabled is Foundations [S3.5](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards-fsbp-controls.html#fsbp-s3-5); "S3 buckets should require requests to use Secure Socket Layer". We felt that it's a low risk issue, is not applicable in all cases, can be auto remediated, and is inconvenient to add to every single S3 bucket.
```

Comment:
`and is inconvenient to add to every single S3 bucket`

you could just call it out and say "requires every bucket to have a bucket policy". inconvenient leaves it hanging whether the technology makes it awkward or we just found it too onerous (or admittedly both).

Reply:
I went with the word `laborious`, thoughts?

Reply:
`onerous` work too